src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -41,14 +43,16 @@ class Translator extends BaseTranslator
      * @param ContainerInterface $container A ContainerInterface instance
      * @param MessageSelector    $selector  The message selector for pluralization
      * @param array              $loaderIds An array of loader Ids
+     * @param array              $resourceDirs An array of resource directories
      * @param array              $options   An array of options
      *
      * @throws \InvalidArgumentException
      */
-    public function __construct(ContainerInterface $container, MessageSelector $selector, $loaderIds = array(), array $options = array())
+    public function __construct(ContainerInterface $container, MessageSelector $selector, $loaderIds = array(), $resourceDirs = array(), array $options = array())
```

Comment:
Adding the argument in the middle of the list can become an issue for people extending this class.